### PR TITLE
pricing for my-e-car cargo bike

### PR DESCRIPTION
**Pull request by ThorstenFroehlinghaus**
tested in test-ipl

### CODE REVIEW

File: config/my-e-car.json [hunk 1]
```diff
@@ -11,283 +11,155 @@
...
-                    },
-                    {
-                        "start": 100,
-                        "interval": 1,
-                        "rate": 0.22
-                    }
-                ],
-                "plan_id": "mini",
+                "name": "A-e Renault ZOE bis 140km - Tag pro Stunde",
+                "per_min_pricing": [ { "start": 0, "end": 1020, "interval": 60, "rate": 1.50 }, { "start": 1021, "interval": 60, "rate": 0.95 } ],
```
Comment (derhuerst) [inline config/my-e-car.json:28]: Can you revert to the old formatting/indentation style? This way, your commits only contain actual changes, which makes them more understandable later on.

(Maybe there's a good reason that I'm not aware of to change the formatting here. Ignore my comment in this case.)

Comment (ThorstenFroehlinghaus) [reply config/my-e-car.json:28]: I changed the formatting of stadtmobil_* and my-e-car because otherwise the pricing descriptions become very lengthy.

**Comment by hbruch** [comment]
Disadvantage of manually reformatting to a more compact version is, that in future no auto-formatting to remove formatting inconsistencies will be possible. Do you value the compact style higher than consistency?

Another request: could you describe the changes in the CHANGELOG.md, please?

**Comment by ThorstenFroehlinghaus** [comment]
I updated CHANGELOG.md.
I want to use the compact pricing_plan formatting style also for future Cantamen providers, because this simplifies their maintenance.

**Review by hbruch — APPROVED** [review]
I approve.

**Pull request merged**